packages/driver/src/cy/commands/actions/select.ts

```diff
@@ -128,7 +128,7 @@ export default (Commands, Cypress, cy) => {
             values.push(value)
 
             // https://github.com/cypress-io/cypress/issues/24739
-            if (options.$el.find(`option[value="${value}"]`).length > 1) {
+            if (options.$el.find(`option[value="${value.replace(/"/g, '\\\"')}"]`).length > 1) {
```

Comment:
@davidfurey Only if you have the spare cycles - but I'm curious as a RegEx noob what this logic actually does and how it resolves the bug. Would appreciate an explanation (but again, only if you have time). Thank you!